### PR TITLE
Always use warp sync for container chains

### PR DESCRIPTION
**Pull request by tmpolaczyk**
Remove the select_sync_mode function which apparently is no longer needed

### CODE REVIEW

File: test/suites/warp-sync/test_warp_sync.ts [hunk 1]
```diff
@@ -260,7 +260,8 @@ describeSuite({
                     "[Container-2000] Warp sync is complete",
                     "[Orchestrator] Detected assignment for container chain 2000",
                     "[Orchestrator] Loaded chain spec for container chain 2000",
-                    "[Orchestrator] Container chain sync mode: Full",
+                    "[Orchestrator] Container chain sync mode: Warp",
+                    "[Container-2000] Can't use warp sync mode with a partially synced database. Reverting to full sync mode.",
```
[the remark this answers is not in the document]
Comment (girazoki) [reply test/suites/warp-sync/test_warp_sync.ts:264]: I am fine leaving it like this

**Review by girazoki — APPROVED** [review]
Just to be clear, we are approving and merging this because @tmpolaczyk detected that the behavior is not broken anymore, but we cannot identify the PR that resolved it. Just putting it here for future reference

File: test/suites/warp-sync/test_warp_sync.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (girazoki) [reply test/suites/warp-sync/test_warp_sync.ts:264]: disregard

**Comment by tmpolaczyk** [comment]
I also tried to set `--sync=warp` to the relay validators, and that didn't work, with this error:

```
WARN tokio-runtime-worker peerset: Report 12D3KooWB1MbMi2KEo9dN2RmZN8GzUTPJWhnz8ncZjFFd5nRQv2j: -536870912 to -1602870452. Reason: Bad warp proof. Banned, disconnecting.    
```

So the underlying issue is fixed for parachains, but not for the relay chain.

**Pull request merged**